scripts/run-json2ts.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const json2ts = require('json-schema-to-typescript');
+const path = require('path');
+
+const schemasDir = path.join('rust', 'json-gen', 'schemas');
+const schemaFiles = fs.readdirSync(schemasDir).filter(file => path.extname(file) === '.json');
+
+Promise.all(schemaFiles.map(schemaFile => {
+  const completeName = path.join(schemasDir, schemaFile);
+  console.log(`Processing: ${completeName}`);
+  return json2ts.compileFromFile(completeName, {declareExternallyReferenced: false});
```

Comment:
I thought `declareExternallyReferenced` would help but I guess they're inlined so it didn't help